src/styles/_class-overrides.scss

```diff
@@ -0,0 +1,525 @@
+            @include u-text('bold');
+            padding: 0 units(0.5);
+            margin: units(1.5) units(2);
+          }
+        }
+      }
+    }
+  }
+
+  //header in mobile view
+  .usa-navbar {
+    @include flex-center-space-between();
+    margin: units(4) 0 units(2) 0;
+    //smaller logo size
+    .header__logo {
+      width: units('card');
+      height: auto;
+      padding-left: units(2);
+    }
+    //nav menu button
+    .usa-menu-btn {
+      @include u-bg('secondary');
+    }
+  }
+
+  //languages in nav bar
+  .header__language-nav-container {
+    //position languages at the bottom
+    position: absolute;
+    bottom: 0;
+    @include u-bg('base-dark');
+    padding: units(1) 0;
+    //right align right to left languages
+    .header__language-nav-RtoL {
+      display: flex;
+      align-items: center;
+      justify-content: flex-end;
+      flex-direction: row;
+      flex-wrap: wrap;
+    }
+    //left align left to right languages
+    .header__language-nav-LtoR {
+      display: flex;
+      align-items: center;
+      justify-content: flex-start;
+      flex-direction: row;
+      flex-wrap: wrap;
+    }
+    //language button styling
+    .header__language-nav-items {
+      .usa-button--unstyled {
+        @include u-text('accent-cool-lighter');
+        &:hover {
+          @include u-text('primary');
+          @include u-bg('base-dark');
+        }
+      }
+    }
+  }
+}
+
+//desktop view
+@media (min-width: units('desktop')) {
+  //show language selector button and project title
+  .banner__language-selector,
+  .header__title {
+    display: block;
+  }
+  .usa-nav {
+    margin-bottom: units(2);
+  }
+  .header__logo-title {
+    @include flex-center-space-between();
+  }
+  .usa-navbar {
+    .header__logo {
+      width: units('card-lg');
+      height: auto;
+    }
+  }
+
+  .usa-nav {
+    padding: units(2);
+    .usa-nav__inner {
+      .usa-nav__primary-item {
+        .usa-nav__submenu {
+          @include u-bg('base-darker');
+          .usa-nav__submenu-item {
+            @include u-bg('base-darker');
+            a {
+              @include u-text('accent-cool-lighter');
+              @include u-bg('base-darker');
+              @include u-text('normal');
+            }
+          }
+        }
+        //links in nav menu
+        a {
+          @include u-text('disabled-dark');
+          @include u-bg('accent-cool-lighter');
+          @include u-text('bold');
+          padding: units(2);
+          margin: 0;
+          &:hover {
+            @include u-text('base-darker');
+            @include u-bg('accent-cool-lighter');
+            @include u-text('bold');
+            padding: units(2);
+            margin: 0;
+            border: none;
+          }
+        }
+        .usa-accordion__button[aria-expanded='false'] {
+          @include u-text('disabled-dark');
+          @include u-bg('accent-cool-lighter');
+          @include u-text('bold');
```

Comment:
Mixins can be used here.

Reply:
done

Reply:
There are repetitions in lines 467-471, 473-477, 482-486, 488-492, 500-504, and 506-510, should we leave those?

Reply:
I think we can definitely work on cleaning up repetition once we have all the features and styling we want